acms-react-app/src/Pages/Signup_Page/Signup.js

```diff
@@ -4,27 +4,59 @@ import { faFacebook, faGoogle, faApple } from '@fortawesome/free-brands-svg-icon
 import NavBar from '../../Components/Navigation_Bars/Not_Logged_In/NavBar.js';
 import Footer from '../../Components/Footer/Footer.js';
 import './signup.css';
-
+import axios from 'axios';
 
 
 function Signup(){
 
-    return(
+    function signup(user){
+        axios.post("http://localhost:8080/api/v1/registration",user)
+        .then(res=>{
+            if(res.status === 201){
+                console.log("User has successfully signed up!")
+            }
+        })
```

Comment:
Good use of axios Cris. You made great Front-End progress for the team and it won't go unnoticed.

Reply:
Thank you